habitat_sim/sensors/noise_models/no_noise_model.py

```diff
@@ -21,7 +22,8 @@
 
 
 @registry.register_noise_model(name="None")
-class NoSensorNoiseModel(SensorNoiseModel):
+@attr.s(auto_attribs=True)
+class NoSensorNoiseModel(SensorNoiseModel):  # type: ignore
```

Comment:
It kept complaining about the fact that the constructor required a GPU_ID due to the parent class, but I couldn't override it. 'attr.s' is deprecated anyhow, we really should be just using Python dataclasses instead.